test/e2e/network/ingress.go

```diff
@@ -647,6 +649,78 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 				}
 			})
 		})
+
+		It("should sync endpoints for both Ingress-referenced NEG and standalone NEG [Unreleased]", func() {
+			name := "hostname"
+			scaleAndValidateExposedNEG := func(num int) {
+				scale, err := f.ClientSet.ExtensionsV1beta1().Deployments(ns).GetScale(name, metav1.GetOptions{})
+				Expect(err).NotTo(HaveOccurred())
+				if scale.Spec.Replicas != int32(num) {
+					scale.Spec.Replicas = int32(num)
+					_, err = f.ClientSet.ExtensionsV1beta1().Deployments(ns).UpdateScale(name, scale)
+					Expect(err).NotTo(HaveOccurred())
+				}
+				wait.Poll(10*time.Second, framework.NEGUpdateTimeout, func() (bool, error) {
+					svcList, err := f.ClientSet.CoreV1().Services(ns).List(metav1.ListOptions{})
```

Comment:
I think you can directly Get the service with name, right? No need to list service and get status.